pages/getting-started/deployments.md

```diff
@@ -3,3 +3,35 @@ title: Getting Started with Plural CD
 description: >-
   Guides you can use to get up and running quickly with Plural CD.
 ---
+
+## Overview
+
+This guide is meant to give you an overview of all the materials needed to get started with Plural’s Fleet Management solution.
+This should be considered a living document; feel free to let us know about any gaps in our documentation and we’ll make it a top priority to plug them to ensure as smooth an experience as possible for future customers.
+
+The general process of setting up your instance is pretty straightforward, and we have documentation and supporting materials below to guide each step:
+
+- set up your management cluster which will host the plural console and supporting operators
+- connect your kubernetes clusters you’d like to manage to that management cluster
+- define services, pipelines, and global services to start shipping workloads into your cluster fleet
+
+## Intro Video
+
+We think the easiest way to understand the workflow is to take a look at our intro video; we have an upload of it [here](https://assets.plural.sh/plural-up.mp4) (It is about 14 minutes long).
```

Comment:
I think we actually have a youtube video link now: https://www.youtube.com/watch?v=caDlvskRkAw&ab_channel=Plural

I'm not actually sure how to do video embeds though, there's probably an example elsewhere in the markdown code.

Reply:
it's all youtube videos in the examples, if we have a yt video of that now, that's perfect, that'll work